src/library/driftory.ts

```diff
@@ -1,5 +1,7 @@
 import loadJs from '@dan503/load-js';
 import { Comic } from './Comic.types';
+export * from './Comic.types'
+export * from './openseadragon.types'
```

Comment:
Exporting these from the main file so that they are easier to access. they can all get imported in the one line

```js
import DriftoryViewer, { Comic } from '@driftory/viewer'
```